test/raven.utils.js

```diff
@@ -108,6 +108,23 @@ describe('raven.utils', function() {
       parseStack([], callback);
       parseStack([{lol: 1}], callback);
     });
+
+    it('should extract context from last stack line', function(done){
```

Comment:
How is this test testing the new code that was just added? I don't see anything about looking for `:\` in the frame or a `\` at all. Just a bit confused on what this is attempting to test.